client/question/questionSpec.js

```diff
@@ -90,28 +93,68 @@ describe('question.js', function() {
     });
   });
 
-  describe('RUNTIME_ERROR_FEEDBACK_MESSAGES', function() {
+  describe('FRIENDLY_SYNTAX_ERROR_TRANSLATIONS', function() {
     it('should have the correct keys and valid values', function() {
-      RUNTIME_ERROR_FEEDBACK_MESSAGES.python.forEach(function(error) {
-        expect(error.checker('test')).toBe(false);
-        expect(typeof error.generateMessage(['NameError: name \'key\' is not ',
-          'defined KeyError: blah on line 47 AttributeError: \'key\' ',
-          'object has no attribute \'length\''].join(''))).toEqual('string');
+      FRIENDLY_SYNTAX_ERROR_TRANSLATIONS.python.forEach(function(error) {
+        expect(error.friendlyErrorCheck('test')).toBe(false);
+        // If any getFriendlyErrorText method that takes in an error string
```

Comment:
The "if" clause of this comment seems to be missing one or more words.

Reply:
There were actually too many words :-P. Changing it to:

"If a getFriendlyErrorText method takes in an error string as input, add a test below."

Most getFriendlyErrorText methods do not take in any input; this is a reminder to add a test for those that do.

Reply:
That sounds good. Thanks!